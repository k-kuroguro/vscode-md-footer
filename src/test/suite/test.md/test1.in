# test1

[Link1](http://www.example.com/link1)
`[InlineCode](http://www.example.com/inlinecode)`
```
[CodeBlock1](http://www.example.com/codeblock1)
```
```sh
[CodeBlock2](http://www.example.com/codeblock2)
```
