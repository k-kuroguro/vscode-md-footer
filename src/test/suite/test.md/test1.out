# test1

[Link1][1]
`[InlineCode](http://www.example.com/inlinecode)`
```
[CodeBlock1](http://www.example.com/codeblock1)
```
```sh
[CodeBlock2](http://www.example.com/codeblock2)
```


[1]:http://www.example.com/link1
